proctoring.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
-  spec.add_dependency "rails", "~> 6.0.0", ">= 6.0.0.0"
+  spec.add_dependency "rails", "~> 6.0", ">= 6.0.0.0"
```

Comment:
the second attribute may not be needed
this should suffice. isn't it?

```ruby
spec.add_dependency "rails", "~> 6.0"
```

Reply:
yes it does, removing it now